### ISSUE TITLE
Session Data not saving in Firefox 

### ISSUE THREAD

**Issue by dookah** [JPedro2/WxT-QA-BOT#4]
On the login page the session data doesnt seem to save on firefox, resulting in all calls returning a 401 since no valid token can be made.

**Comment by petmorri** [comment]
It works fine for me in Firefox – can you try again when you get time?

**Comment by dookah** [comment]
Not working on Firefox on MacOS but works on Windows. @JPedro2 can you confirm?

**Comment by JPedro2** [comment]
Yeah it doesnt work on my side either. Also, it's saying "Success" when submitting although there's a 401 response back from the API

<img width="1273" alt="Screenshot 2020-05-11 at 10 14 24" src="https://user-images.githubusercontent.com/22790804/81554511-d6f89880-937e-11ea-8030-a05ed956ecbd.png">

**Comment by petmorri** [comment]
I've been trying to investigate this further but I haven't been able to reproduce the issue on Windows or Linux (I don't have access to Mac). One thing we could try is using [`Window.localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) instead of [`Window.sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) but I won't be able to verify if it works as I can't reproduce the issue.
